### PR TITLE
[WIP] [KEP-4346]: feature-gate InformerMetrics to alpha

### PR DESCRIPTION
**Pull request by chenk008**
add feature description for InformerMetrics alpha promotion

KEP:  https://github.com/kubernetes/enhancements/issues/4346

code PR:  https://github.com/kubernetes/kubernetes/pull/123367

### CODE REVIEW

File: content/en/docs/reference/command-line-tools-reference/feature-gates/informer_metrics.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+_build:
+  list: never
+  render: false
+
+stages:
+  - stage: alpha 
+    defaultValue: false
+    fromVersion: "1.30"
+---
+Enable support for exposing informer/reflector metrics.
```
Comment (sftim) [inline content/en/docs/reference/command-line-tools-reference/feature-gates/informer_metrics.md:13]: My suggestion: add _informer_ and _reflector_ into the [Kubernetes glossary](https://kubernetes.io/docs/reference/glossary/), and then refer to those using [glossary shortcodes](https://kubernetes.io/docs/contribute/style/hugo-shortcodes/#glossary).

**Not needed for the alpha**

**Comment by drewhagen** [comment]
Hello @chenk008 👋 please take a look at [Documenting for a release - PR Ready for Review](https://kubernetes.io/docs/contribute/new-content/new-features/#pr-ready-for-review) to get your PR ready for review before Tuesday March 12th 2024 18:00 PST. Thank you!

**Comment by chenk008** [comment]
@drewhagen Sorry, i think the code PR is still under discussion, so it will delay.

**Comment by drewhagen** [comment]
/hold
@chenk008 Kubernetes 1.30 is releasing today.
This PR will need to be targeting the `dev-1.31` branch once it is available. Our team will keep you posted.

**Comment by drewhagen** [comment]
You can reopen this in the next release cycle and target `dev-1.31`. Thanks! :)

**Comment by drewhagen** [comment]
Heads up that `dev-1.31` branch is created now, if you choose to reopen this for the next release.

**Pull request closed without merging**